packages/attest/src/attest.ts

```diff
@@ -28,6 +28,8 @@ export type AttestOptions = {
   // Sigstore instance to use for signing. Must be one of "public-good" or
   // "github".
   sigstore?: SigstoreInstance
+  // HTTP headers to include in request to attestations API.
+  headers?: {[header: string]: string | number | undefined}
```

Comment:
Should we have a block list to make sure the client does not set a header that may interfere with any standard headers that are sent?

Reply:
I looked into this and everything we'd maybe want to block (`user-agent`, `accept`, `content-type`, `authorization`) is already overridden in the `@octokit/request` library. I think we're good to pass-along whatever and let octokit sort it out.